README.md

```diff
@@ -107,7 +107,7 @@ See [the examples](./examples/README.md) for more detailed usage or read about a
 * `customDiffDir`: A custom absolute path of a directory to keep this diff in
 * `customSnapshotIdentifier`: A custom name to give this snapshot. If not provided one is computed automatically. When a function is provided it is called with an object containing `testPath`, `currentTestName`, `counter` and `defaultIdentifier` as its first argument. The function must return an identifier to use for the snapshot.
 * `diffDirection`: (default: `horizontal`) (options `horizontal` or `vertical`) Changes diff image layout direction
-* `noColors`: (default `false`) Removes coloring from console output, useful if storing the results in a file
+* `noColors`: If set to `true`/`false` explicitly sets coloring from console output, useful if storing the results in a file
```

Comment:
Im not sure this new wording is more clear. Can you leave it as is?

Reply:
Ok, I think it should be so, since I changed the default value:
```suggestion
* `noColors`: Removes coloring from console output, useful if storing the results in a file
```
Does it look good?

Reply:
Yes!